run_aprime.bash

```diff
@@ -0,0 +1,325 @@
+#!/bin/bash
+
+#
+# Template driver script to generate coupled diagnostics on edison
+#
+# Basic usage:
+#       1. copy this template to something like run_aprime_$user.bash
+#       2. open run_aprime_$user.bash and set user defined, case-specific variables
+#       3. execute: ./run_aprime_$user.bash 
+
+# Meaning of acronyms/words used in variable names below:
+#	test:		 Test case
+#	ref:		 Reference case or 'obs'
+#	ts: 		 Time series; e.g. test_begin_yr_ts, here ts refers to time series
+#       climateIndex_ts: Time series for climate indexes such as Nino3.4.
+#                        NB: if begin_yr_climateIndex_ts=1 and end_yr_climateIndex_ts=9999, the
+#                        climate index time series is computed over the full available data set
+#	climo: 		 Climatology
+#	begin_yr: 	 Model year to start analysis 
+#	end_yr:		 Model year to end analysis
+#       condense:        Create a new file for each variable with time series data for that variable.
+#                        This is used to create climatology (if not pre-computed) and in generating time series plots
+#	archive_dir:	 Location of model generated output directory
+#	scratch_dir:	 Location of directory where the user wants to store files generated by the diagnostics.
+#			 This includes climos, remapped climos, condensed files and data files used for plotting. 
+#	short_term_archive:   Adds /atm/hist after the casename. If the data sits in a different structure, add it after
+#	                      the casename in test_casename 
+
+# USER DEFINED CASE SPECIFIC VARIABLES TO SPECIFY (REQUIRED)
+
+# Main directory where all analysis output is stored
+# (e.g., plots will go in a specific subdirectory of $output_base_dir,
+# as will log files, generated climatologies, etc)
+export output_base_dir=/dir/to/analysis/output
+
+# MACHINE SPECIFIC VARIABLES (no need to modify these in most cases)
+machname4=`echo $HOSTNAME | cut -c1-4`
+machname5=`echo $HOSTNAME | cut -c1-5`
+machname6=`echo $HOSTNAME | cut -c1-6`
+if [ $machname6 == "edison" ]; then # NERSC
```

Comment:
I would do this differently.  I would get rid of `machname#` and instead do:
```bash
if [ ${HOSTNAME:0:6} == "edison" ]; then
  machname="nersc"
elif [ ${HOSTNAME:0:4} == "rhea" ] || [ ${HOSTNAME:0:5} == "titan" ]; then
  machname="olcf"
elif [ ${HOSTNAME:0:5} == "aims4" ] || [ ${HOSTNAME:0:5} == "acme1" ]; then
  machname="llnl"
elif [ ${HOSTNAME:0:4} == "wolf" ]; then
  machname="lanl"
else
  echo "Unsupported host $HOSTNAME. Exiting."
  exit 1
if
```
It might make sense to use something like `centername` or `hpcname` instead of `machname` if you go this way. Then, I would use `machname` (or whatever you decide to call it) from here on (without 4, 5, 6).  From what I could see, there's no need to distinguish rhea from titan or aims4 from acme1 but keep those names separate if so.